pkg/network/node/node.go

```diff
@@ -369,7 +369,9 @@ func (node *OsdnNode) Start() error {
 		if err := node.SetupEgressNetworkPolicy(); err != nil {
 			return err
 		}
-		if err := node.egressIP.Start(node.networkInformers, node.nodeIPTables); err != nil {
+		nodeinformer := node.kubeInformers.Core().V1().Nodes()
+		nodeinformer.Informer().GetController()
```

Comment:
what?
(Docs for "GetController": "`// GetController is deprecated, it does nothing useful`)

also, maybe better to just pass `node.kubeInformers` to `Start` and let it pick out the specific informers it needs, rather than having node.go know

Reply:
This is also something left over from my testing, I had some hick-ups while writing this and saw this comment: https://github.com/openshift/sdn/blob/46fd1cd3346980b0acc869e3dfdc8e6c816f7de2/pkg/network/master/master.go#L79-L84 thinking it might solve my problem. 

I'll remove that.

Reply:
lol. yay crufty code